tests/Unit/Evolution/DgSubcell/Test_TwoMeshRdmpTci.cpp

```diff
@@ -0,0 +1,127 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#include "Framework/TestingFramework.hpp"
+
+#include <array>
+#include <cstddef>
+#include <limits>
+
+#include "DataStructures/DataBox/Tag.hpp"
+#include "DataStructures/DataVector.hpp"
+#include "DataStructures/Tensor/Tensor.hpp"
+#include "DataStructures/Variables.hpp"
+#include "Domain/LogicalCoordinates.hpp"
+#include "Evolution/DgSubcell/Projection.hpp"
+#include "Evolution/DgSubcell/Tags/Inactive.hpp"
+#include "Evolution/DgSubcell/TwoMeshRdmpTci.hpp"
+#include "NumericalAlgorithms/Spectral/Mesh.hpp"
+#include "NumericalAlgorithms/Spectral/Spectral.hpp"
+#include "Utilities/Gsl.hpp"
+#include "Utilities/MakeArray.hpp"
+#include "Utilities/TMPL.hpp"
+
+namespace {
+namespace Tags {
+struct Scalar : db::SimpleTag {
+  using type = ::Scalar<DataVector>;
+};
+
+template <size_t Dim>
+struct Vector : db::SimpleTag {
+  using type = tnsr::I<DataVector, Dim>;
+};
+}  // namespace Tags
+
+template <size_t Dim>
+DataVector soln(const tnsr::I<DataVector, Dim, Frame::Logical>& coords,
+                const size_t add_discontinuity_at) noexcept {
+  DataVector result =
+      Spectral::compute_basis_function_value<Spectral::Basis::Legendre>(
+          1, get<0>(coords));
+  for (size_t d = 1; d < Dim; ++d) {
+    result += Spectral::compute_basis_function_value<Spectral::Basis::Legendre>(
+        1, coords.get(d));
+  }
+
+  if (add_discontinuity_at > 0) {
```

Comment:
change `add_discontinuity_at` to a bool since that is how it is used; then you don't need ternary operators at the call site...